Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM python:3.6
```

Comment:
No need to create a virtual environment inside docker. Use `pip install -r requirements.txt`